.resinci.yml

```diff
@@ -0,0 +1,11 @@
+---
+reviewers: 1
+docker:
+  publish: false
+  builds:
+    - dockerfile: Dockerfile
+      path: .
+      docker_repo: balena/amd64-balena-multibuild-scripts
+      publish: false
+npm:
+  run: false
```

Comment:
I've copied this `.resinci.yml` file from `balena-supervisor`, but this needs discussion. Very much a draft.